app/models/auction_query.rb

```diff
@@ -47,9 +45,16 @@ def complete_and_successful
       .paid
   end
 
-  def rejected
+  def completed
```

Comment:
I also felt like this name wasn't quite right....also not sure this is the exact query we should use for all the stats. the basic thing I was looking for in some was "over with bids" but on some cases just "over" would be fine....